First Commit

cccmir opened the pull request:
Added Logging using serilog
added Scraper and scraping service

cccmir committed:
First Commit

Added Logging using serilog
added Scraper and scraping service

ShaharShokraniEshetTours commented on CalendarEventCore/CalendarEvents.Services/GenericService.cs:
```diff
@@ -0,0 +1,141 @@
+            }
+            catch (Exception ex)
+            {
+                _log.LogError(ex, "Failed inserting object to DB");
```
@cccmir Isn't it better to put the LogError from inside the `ResultService.Fail(ex);`?

cccmir replied:
you probably right, i added the logging to quickly just to see the flow

cccmir committed:
Added Docker suuport for webApi and sqlserverDb

cccmir committed:
added rect simple app to view event

cccmir committed:
added db connection string vi enviorment var when using docker

cccmir committed:
added docker improvments:
aspnet app will wait until DB is up
connectionstring will come from Env Var
adde .env file for env vars

cccmir commented:
@ShaharShokrani added docker support and small react app with table of events
